app/views/step_by_step_pages/publish_or_delete.html.erb

```diff
@@ -31,8 +31,10 @@
           <p>Before publishing check for broken links.</p>
           <p>After publishing add this step by step to a mainstream browse category and a taxonomy topic.</p>
           <%= link_to 'Publish changes', step_by_step_page_publish_path(@step_by_step_page), class: "btn btn-primary publish-button" %>
-          <p>Or</p>	
-          <%= link_to 'Schedule publish', step_by_step_page_schedule_path(@step_by_step_page), class: "btn btn-info" %>	
+          <p>Or</p>
```

Comment:
This "Or" needs to be wrapped in the guard clause as well, otherwise users without the scheduling permission are shown:

![Screen Shot 2019-07-24 at 17 51 26](https://user-images.githubusercontent.com/5793815/61812706-f2200900-ae3b-11e9-9587-e6d8f3ae78b9.png)